Simplenote/src/main/java/com/automattic/simplenote/utils/CrashUtils.java

```diff
@@ -0,0 +1,114 @@
+package com.automattic.simplenote.utils;
+
+import android.content.Context;
+import android.content.res.Resources;
+import android.os.Build;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+
+import com.automattic.android.tracks.CrashLogging.CrashLogging;
+import com.automattic.android.tracks.CrashLogging.CrashLoggingDataProvider;
+import com.automattic.android.tracks.TracksUser;
+import com.automattic.simplenote.BuildConfig;
+import com.automattic.simplenote.Simplenote;
+import com.simperium.client.User;
+
+import java.util.Locale;
+import java.util.Map;
+
+public class CrashUtils implements CrashLoggingDataProvider {
+
+    private User currentUser;
+    private Locale locale;
+
+    private static CrashUtils sharedInstance;
+
+    public static void initWithContext(Context context) {
+
+        if (sharedInstance != null) {
+            return;
+        }
+
+        sharedInstance = new CrashUtils();
+        sharedInstance.locale = localeForContext(context);
+
+        CrashLogging.start(context, sharedInstance);
+    }
+
+    public static void setCurrentUser(User user) {
+        sharedInstance.currentUser = user;
+        CrashLogging.setNeedsDataRefresh();
+    }
+
+    public static void clearCurrentUser() {
+        sharedInstance.currentUser = null;
+        CrashLogging.setNeedsDataRefresh();
+    }
+
+    @Override
+    public String sentryDSN() {
+        return BuildConfig.SENTRY_DSN;
```

Comment:
This doesn't seem to work well with what we have in the secrets repository. The entry there is "sentryDSN" which would translate to SENTRY_D_S_N here. 
The entry looks correct in the example file here, so I guess you want to change the file in the repo.

Reply:
🤦‍♂ – you're totally right! Had to rework the BuildConfig logic when merging with `develop` and didn't add it to the store. Updated the PR with the new commit hash and tested this all again from scratch. Should be good now.

Sorry about that!